grpc/node-cubic/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@cobaltspeech/sdk-cubic",
+  "version": "1.0.1",
+  "description": "",
+  "main": "dist/sdk-wj.umd.js",
+  "module": "dist/sdk-wj.es5.js",
+  "typings": "dist/types/sdk-wj.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc --module commonjs && rollup -c rollup.config.ts",
+    "start": "rollup -c rollup.config.ts -w",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "gen": "protoc -I . --plugin=./node_modules/.bin/protoc-gen-ts_proto --ts_proto_out=./src/grpc ../cubic.proto"
+  },
+  "keywords": [],
+  "author": "Cobalt Speech And Language, Inc.",
+  "license": "Apache",
```

Comment:
Our top level license says Apache v2.0.  Does that matter here?  Would NodeJS treat this as Apache v1.0 instead of 2.0?